src/app/products/products-mix/create-product-mix/create-product-mix.component.ts

```diff
@@ -0,0 +1,92 @@
+/** Angular Imports */
+import { Component, OnInit } from '@angular/core';
+import { FormGroup, FormBuilder, Validators, FormControl } from '@angular/forms';
+import { Router, ActivatedRoute } from '@angular/router';
+
+/** Custom Services */
+import { ProductsService } from '../../products.service';
+
+/**
+ * Create Product mix component.
+ */
+@Component({
+  selector: 'mifosx-create-product-mix',
+  templateUrl: './create-product-mix.component.html',
+  styleUrls: ['./create-product-mix.component.scss']
+})
+export class CreateProductMixComponent implements OnInit {
+
+  /** Product mix form. */
+  productMixForm: FormGroup;
+  /** Products mix template data. */
+  productsMixTemplateData: any;
+  /** Product option data. */
+  productOptionData: any;
+  /** Product data. */
+  productData: any;
+
+  /**
+   * Retrieves the Products mix template data from `resolve`.
+   * @param {FormBuilder} formBuilder Form Builder.
+   * @param {ProductsService} productsService Products Service.
+   * @param {ActivatedRoute} route Activated Route.
+   * @param {Router} router Router for navigation.
+   */
+  constructor(private formBuilder: FormBuilder,
+              private productsService: ProductsService,
+              private route: ActivatedRoute,
+              private router: Router) {
+              this.route.data.subscribe(( data: { productsMixTemplate: any }) => {
+      this.productsMixTemplateData = data.productsMixTemplate;
```

Comment:
I will approve the task for now, just make sure you correctly indent here though before we merge.